src/sentry/incidents/endpoints/organization_alert_rule_index.py

```diff
@@ -121,10 +121,15 @@ def create_metric_alert(self, request, organization, project=None):
                 return Response({"uuid": client.uuid}, status=202)
                 # TODO handle slack channel lookup AND anomaly detection
             else:
+                http_status = 201
                 alert_rule = serializer.save()
                 if alert_rule.detection_type == AlertRuleDetectionType.DYNAMIC.value:
                     resp = send_historical_data_to_seer(alert_rule=alert_rule, user=request.user)
```

Comment:
Can we label this something more descriptive rather than `resp`?
Was a bit confused since this isn't the straight Seer response either, but a modified response

Might be out of scope for this pr though :/ 
since we're only using it to determine whether the seer backfill was successful or not, we could just have `send_historical_data_to_seer` raise an exception if there's not enough data instead 🤔